scss/_dropdown.scss

```diff
@@ -10,45 +10,23 @@
 
 .dropdown-toggle {
   // scss-docs-start dropdown-toggle-css-vars
-  --#{$prefix}dropdown-toggle-padding-x: #{$dropdown-padding-x}; // Boosted mod
-  --#{$prefix}dropdown-toggle-color: #{$dropdown-color}; // Boosted mod
-  --#{$prefix}dropdown-toggle-bg: #{$dropdown-bg}; // Boosted mod
-  --#{$prefix}dropdown-toggle-border-color: #{$dropdown-border-color}; // Boosted mod
+  --#{$prefix}btn-padding-x: #{$dropdown-padding-x}; // Boosted mod
   // scss-docs-end dropdown-toggle-css-vars
 
-  // Boosted mod
-  padding-right: var(--#{$prefix}dropdown-toggle-padding-x);
-  padding-left: var(--#{$prefix}dropdown-toggle-padding-x);
-  color: var(--#{$prefix}dropdown-toggle-color);
   white-space: nowrap;
-  background-color: var(--#{$prefix}dropdown-toggle-bg);
-  border-color: var(--#{$prefix}dropdown-toggle-border-color);
-
-  .bg-dark &:not(.dropdown-toggle-split),
-  .navbar-dark &:not(.dropdown-toggle-split) {
-    // scss-docs-start dropdown-toggle-dark-variant-css-vars
-    --#{$prefix}dropdown-toggle-color: #{$dropdown-bg};
-    --#{$prefix}dropdown-toggle-bg: #{$dropdown-color};
-    --#{$prefix}dropdown-toggle-border-color: #{$gray-700};
-    --#{$prefix}dropdown-toggle-border-hover-color: #{$dropdown-link-hover-color};
-    // scss-docs-end dropdown-toggle-dark-variant-css-vars
-
-    color: var(--#{$prefix}dropdown-toggle-color);
-    background-color: var(--#{$prefix}dropdown-toggle-bg);
-    border-color: var(--#{$prefix}dropdown-toggle-border-color);
-
-    &:hover {
-      border-color: var(--#{$prefix}dropdown-toggle-border-hover-color);
-    }
-  }
-
-  .btn-group &:not(.btn-inverse) {
-    border-color: var(--#{$prefix}dropdown-toggle-color); // Boosted mod
-  }
-  // End mod
 
   // Generate the caret automatically
   @include caret();
+
+  // Boosted mod
+  &:not(.dropdown-toggle-split) {
```

Comment:
If this is accepted, should maybe be a button variant directly!

Reply:
I'll let you create a detailed "enhancement" issue after this PR then.

Reply:
I'll let you create a detailed "enhancement" issue after this PR then.